### ISSUE TITLE
Copying commit has crashes Lazygit

### ISSUE THREAD

**Issue by MuratUrsavas** [jesseduffield/lazygit#1114]
**Describe the bug**
I've tried to copy the hash from the reflog window via "CTLR + o" but got this message:
```
2020/12/14 14:21:13 An error occurred! Please create an issue at https://github.com/jesseduffield/lazygit/issues

*errors.errorString No clipboard utilities available. Please install xsel, xclip, wl-clipboard or Termux:API add-on for termux-clipboard-get/set.
github.com/jesseduffield/lazygit/main.go:133 (0xa09eb3)
runtime/proc.go:203 (0x43149e)
runtime/asm_amd64.s:1357 (0x45cec1)
```

Got fixed after installing xsel. I guess xsel or xclip should be a dependency. 

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Lazygit Version : build date=2020-10-14, build source=debian, version=0.23.6, os=linux, arch=amd64

**Comment by jesseduffield** [comment]
I think that error message is appropriate, but we should have it appear in an error panel rather than have it crash the app. This should be simple enough to implement so I'll put a hacktoberfest label on it.

**Comment by MuratUrsavas** [comment]
I think adding 'xsel' to the dependencies could be a good idea. It would get rid of this error in the first place.

**Comment by jesseduffield** [comment]
when you say dependencies are you referring specifically to the ubuntu installation method? I am not super across how that's set up :/

**Comment by MuratUrsavas** [comment]
Yes, exactly. It's done in the deb configuration. Not a big deal. I've learned it recently but need some refresh as some time passed since. Will check it out and return back.

**Comment by MuratUrsavas** [comment]
Got back: You just need to add xsel into the control file of debian package into the "Build-Depends:" section. This way apt would know it should install xsel before lazygit.

**Comment by jesseduffield** [comment]
Now we simply need to work out how to access that control file haha

**Comment by MuratUrsavas** [comment]
:) 

Who's running the launchpad PPA?

**Comment by Tej12345** [comment]
Also if you get **status -1** when trying to copy commit hash via **ssh**.
use **-X** flag when connecting to your server.
```
ssh -X my_ssh_server
```
